plugins/pencilblue/controllers/admin/plugins/plugin_settings.js

```diff
@@ -97,6 +98,12 @@ module.exports = function(pb) {
                     }
                 }
 
+                // First sort the settings alphabetically, then by heading/group
+                clone.sort(sortOn("displayName"));
+                clone.sort(sortOn("heading"));
+
+                insertGroupHeadings(clone);
```

Comment:
Based on the recommended changes for this function this line will need to update to:
```clone = PluginSettingsFormController.insertGroupHeadings(clone);```